.Rproj.user/54881094/bibliography-index/biblio-files

```diff
@@ -0,0 +1,2 @@
+1593687486:C:/Users/Jonee/Documents/GitHub/jasp-vp-test/book.bib
```

Comment:
delete this file (at least on the remote), and add `.Rproj.user` to .gitignore? In fact, I'd copy most of https://github.com/jasp-stats/jaspDescriptives/blob/master/.gitignore to .gitignore.